source/dotnet/wholesale-api/Edi/Validation/AggregatedTimeSeries/Rules/MeteringPointTypeValidationRule.cs

```diff
@@ -30,7 +30,11 @@ public class MeteringPointTypeValidationRule : IValidationRule<AggregatedTimeSer
 
     public Task<IList<ValidationError>> ValidateAsync(AggregatedTimeSeriesRequest subject)
     {
-        if (IsValidMeteringPointType(subject.MeteringPointType)) return Task.FromResult(NoError);
+        if (subject.MeteringPointType == string.Empty)
+            return Task.FromResult(NoError);
+
+        if (IsValidMeteringPointType(subject.MeteringPointType))
+            return Task.FromResult(NoError);
 
         return Task.FromResult(InvalidMeteringPointType);
     }
```

Comment:
With the new validation rule, is the "ValidationError" then still correct, since now string.Empty is also valid?

It feels wrong to validate something in the method IsValidMeteringPointType, but not everything. Maybe we could refactor it like:
```suggestion
    public Task<IList<ValidationError>> ValidateAsync(AggregatedTimeSeriesRequest subject)
    {
        if (IsValidMeteringPointType(subject.MeteringPointType))
            return Task.FromResult(NoError);

        return Task.FromResult(InvalidMeteringPointType);
    }

    private static bool IsValidMeteringPointType(string meteringPointType)
    {
        return
            meteringPointType == string.Empty
            || _validMeteringPointTypes.Contains(meteringPointType, StringComparer.OrdinalIgnoreCase);
    }    
```